frontend/src/types.ts

```diff
@@ -1025,25 +1048,42 @@ export interface BillingType {
 
 export type BillingVersion = 'v1' | 'v2'
 
+export interface BillingV2FeatureType {
+    key: string
+    name: string
+    description?: string
+    unit?: string
+    limit?: number
+    note?: string
+    group?: AvailableFeature
+}
+
+export interface BillingV2TierType {
+    unit_amount_usd: string
+    current_amount_usd?: string | null
+    up_to: number | null
+}
+
 export interface BillingProductV2Type {
-    type: 'EVENTS' | 'RECORDINGS'
+    type: 'EVENTS' | 'RECORDINGS' | 'ENTERPRISE' | 'BASE' | 'events' | 'recordings' | 'enterprise' | 'base'
```

Comment:
Why do we need these upper and lower case?

Reply:
Because we are reusing existing types and extending them to work with the new plans API. The billing service returns products with uppercased product types, but the plans API returns them lowercase. Could update the billing service to always return lowercase, but I just made it work for now.

Reply:
I'm actually not seeing the uppercase ones any more so I just changed to all lower